splice_machine/src/main/java/com/splicemachine/derby/utils/Vacuum.java

```diff
@@ -130,8 +158,8 @@ public void vacuumDatabase(long oldestActiveTransaction) throws SQLException{
                             requiresDroppedId = true;
                         }
                     }
-                    if (!ignoreDroppedId && table.getDroppedTransactionId() != null) {
-                        TxnView txn = SIDriver.driver().getTxnSupplier().getTransaction(Long.parseLong(table.getDroppedTransactionId()));
+                    if (!ignoreDroppedId && droppedConglomerates.containsKey(tableConglom)) {
```

Comment:
It seems that for tables that are dropped between 1838 and 1840, since they are not registered in  the table DROPPED_CONGLOMERATES, they won't get a chance to be vacuumed. 
maybe the code to check table.getDroppedTransactionId() should still be retained?

Reply:
Added code to handle that case, good catch! I also added an IT on VaccumIt specifically for this case.

Thanks for the review!